parts/kubernetesagentcustomdataforcoreos.yml

```diff
@@ -0,0 +1,120 @@
+#cloud-config
```

Comment:
CoreOS  cloudconfig  is different than normal CloudConfig. For Example, Packages are not valid config when targeting CoreOS. I also propose to keep it separate such that this will eventually Ignition based config as CoreOS is going that way.